src/control/lib/ipmctl/nvm_ctest_linux_amd64.go

```diff
@@ -1,5 +1,5 @@
 //
-// (C) Copyright 2018-2022 Intel Corporation.
+// (C) Copyright 2022 Intel Corporation.
```

Comment:
Looks like some unbidden copyright changes?

Reply:
but the file didn't exist until 2022